msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -137,6 +137,117 @@ async def on_message_activity(self, turn_context: TurnContext):
+    # [App manifest v1.11 or earlier](#tab/app-manifest-v111-or-earlier)
+    
+    ```json
+    
+     “webApplicationInfo”: {
+    
+         "id": "123456c8-67d2-4f54-b74e-408b195c4cbc",
+         "resource": "https: //AnyString",
+         "applicationPermissions": [
+             "ChatMessageReadReceipt.Read.Chat"
+         ]
+     },
+    
+    ```
+    
+    ---
+    
+    You can also add RSC permissions through Graph API. For more information, see [`consentedPermissionSet`](/graph/api/userteamwork-teamsappinstallation-upgrade#http-request).
+
+* Override the method `OnTeamsReadReceiptAsync` with `IsMessageRead` handler.
+
+  The `IsMessageRead` helper method is useful to determine if the message is read by the recipients. If the `compareMessageId` is less than or equal to the `LastReadMessageId`, then the message has been read. Override the `OnTeamsReadReceiptAsync` method to receive read receipts with [`IsMessageRead`](/dotnet/api/microsoft.bot.schema.teams.readreceiptinfo.ismessageread#microsoft-bot-schema-teams-readreceiptinfo-ismessageread(system-string)) helper method:
+
+    ```csharp
+    
+    protected override async Task OnTeamsReadReceiptAsync(ReadReceiptInfo readReceiptInfo, ITurnContext<IEventActivity> turnContext, CancellationToken cancellationToken) 
+    
+    ```
+
+    Following is an example of read receipts event request that a bot receives:
+    
+    ```json
+    {
+        "name": "application/vnd.microsoft.readReceipt",
+        "type": "event",
+        "timestamp": "2023-08-16T17:23:11.1366686Z",
+        "id": "f:b4783e72-9d7b-2ed9-ccef-ab446c873007",
+        "channelId": "msteams",
+        "serviceUrl": "https://canary.botapi.skype.com/amer/",
+        "from": {
+            "id": "29:1-8Iuh70W9pRqV8tQK8o2nVjxz33RRGDKLf4Bh7gKnrzN8s7e4vCyrFwjkPbTCX_Co8c4aXwWvq3RBLr-WkkVMw",
+            "aadObjectId": "5b649834-7412-4cce-9e69-176e95a394f5"
+        },
+        "conversation": {
+            "conversationType": "personal",
+            "tenantId": "528dbe3f-16e0-4e37-84a2-00cc305847ed",
+            "id": "a:1xlimp68NSUxEqK0ap2rXuwC9ITauHgV2M4RaDPkeRhV8qMaFn-RyilMZ62YiVdqs8pp43yQaRKvv_U2S2gOS5nM-y_pOxVe4BW1qMGPtqD0Bv3pw-nJXF0zhDlZHMZ1Z"
+        },
+        "recipient": {
+            "id": "28:9901a8b6-4fef-428b-80b1-ddb59361adeb",
+            "name": "Test Bot"
+        },
+        "channelData": {
+            "tenant": {
+                "id": "6babcaad-604b-40ac-a9d7-9fd97c0b779f"
+            }
+        },
+        "value": {
+            "lastReadMessageId": "1692206589131"
+        }
+    }
+    
+    ```
+
+* Read receipt [admin setting](/microsoftteams/messaging-policies-in-teams#messaging-policy-settings) or [user setting](https://support.microsoft.com/office/use-read-receipts-for-messages-in-microsoft-teams-533f2334-32ef-424b-8d56-ed30e019f856) is turned on for the tenant for the bot to receive the read receipt events. The tenant admin or the user can enable or disable the read receipt setting.
+
```

Comment:
This is confusing. I don't think we have a separate  supported property called `ReadStatus`. If the intention of this line is that a bot developer can count the number of events to measure the engagement, let's just mention that directly without using ReadStatus property.

Reply:
Updated